### PR TITLE
[Bug 1924249]Make version information easy to copy/paste Edit

### PR DESCRIPTION
**Pull request by yaminb**
1. Make about dialog resizeable.
2. added copy version info button to version info screen
3. when pressed, this button copies the version info table into the clipboard as a csv

pinta_new_about_copy_vinfo

//sample paste from the button
Title,Version,Path
atk-sharp,2.12.0.0,/usr/lib/mono/gac/atk-sharp/2.12.0.0__35e10195dab3c99f/atk-sharp.dll
gdk-sharp,2.12.0.0,/usr/lib/mono/gac/gdk-sharp/2.12.0.0__35e10195dab3c99f/gdk-sharp.dll
glib-sharp,2.12.0.0,/usr/lib/mono/gac/glib-sharp/2.12.0.0__35e10195dab3c99f/glib-sharp.dll
gtk-sharp,2.12.0.0,/usr/lib/mono/gac/gtk-sharp/2.12.0.0__35e10195dab3c99f/gtk-sharp.dll
Mono.Addins,1.0.0.0,/home/yamin/develop/Pinta/bin/Mono.Addins.dll
Mono.Addins.Setup,1.0.0.0,/home/yamin/develop/Pinta/bin/Mono.Addins.Setup.dll
Mono.Cairo,4.0.0.0,/usr/lib/mono/gac/Mono.Cairo/4.0.0.0__0738eb9f132ed756/Mono.Cairo.dll
Mono.Posix,4.0.0.0,/usr/lib/mono/gac/Mono.Posix/4.0.0.0__0738eb9f132ed756/Mono.Posix.dll
mscorlib,4.0.0.0,/usr/lib/mono/4.5/mscorlib.dll
pango-sharp,2.12.0.0,/usr/lib/mono/gac/pango-sharp/2.12.0.0__35e10195dab3c99f/pango-sharp.dll
Pinta,1.8.0.0,/home/yamin/develop/Pinta/bin/Pinta.exe
Pinta.Core,1.8.0.0,/home/yamin/develop/Pinta/bin/Pinta.Core.dll
Pinta.Docking,1.8.0.0,/home/yamin/develop/Pinta/bin/Pinta.Docking.dll
Pinta.Effects,1.8.0.0,/home/yamin/develop/Pinta/bin/Pinta.Effects.dll
Pinta.Gui.Widgets,1.8.0.0,/home/yamin/develop/Pinta/bin/Pinta.Gui.Widgets.dll
Pinta.Resources,1.8.0.0,/home/yamin/develop/Pinta/bin/Pinta.Resources.dll
Pinta.Tools,1.8.0.0,/home/yamin/develop/Pinta/bin/Pinta.Tools.dll
System,4.0.0.0,/usr/lib/mono/gac/System/4.0.0.0__b77a5c561934e089/System.dll
System.Configuration,4.0.0.0,/usr/lib/mono/gac/System.Configuration/4.0.0.0__b03f5f7f11d50a3a/System.Configuration.dll
System.Core,4.0.0.0,/usr/lib/mono/gac/System.Core/4.0.0.0__b77a5c561934e089/System.Core.dll
System.Xml,4.0.0.0,/usr/lib/mono/gac/System.Xml/4.0.0.0__b77a5c561934e089/System.Xml.dll

### CODE REVIEW

File: Pinta/Dialogs/VersionInformationTabPage.cs [hunk 1]
```diff
@@ -90,5 +107,55 @@ protected override void OnDestroyed ()
 
 			base.OnDestroyed ();
 		}
+
+		void CopyButton_Clicked (object sender, EventArgs e)
+		{
+			String delimeter = ",";
+			String linesep = "\n";
```
Comment (cameronwhite) [inline Pinta/Dialogs/VersionInformationTabPage.cs:114]: Should this be using `Environment.NewLine` for the platform-specific line separator? (I don't actually know if the CSV spec has a requirement either way)

**Review by cameronwhite — APPROVED** [review]
Looks good, thanks! Just have the one question in the comments

**Comment by yaminb** [comment]
I can change it to environment.newline. probably makes more sense for
Windows users.

Makes it easier to paste into notepad or whatever.

File: Pinta/Dialogs/VersionInformationTabPage.cs (hunk 1 above)
Comment (yaminb) [reply Pinta/Dialogs/VersionInformationTabPage.cs]: done

**Review by cameronwhite — APPROVED** [review]

**Pull request merged**